code/modules/jobs/job_types/ai.dm

```diff
@@ -10,7 +10,7 @@
 	supervisors = "your laws"
 	req_admin_notify = TRUE
 	minimal_player_age = 30
-	exp_requirements = 240
+	exp_requirements = 1200
```

Comment:
good god thats near impossible considering you can only get the exp by playing silicon.

Reply:
Being even a half decent AI requires a strong understanding of silicons and obeying your laws, as well as frequently giving you multiple borg bodies to control.

I'm not unmoving in believing 20h is excessive, but I think it's reasonable given the burden that an AI is expected to carry.

Reply:
4 hours as silicon?

Reply:
Currently four hours is all we require before giving players access to what is one small step down from admin permissions - I don't think it should be less than at least ten personally.

Reply:
Adjusted down to 15 since getting silicon hours is somewhat gated, although you can always ask to be borged as an assistant or such.